.github/workflows/ci-win.yml

```diff
@@ -182,21 +182,6 @@ jobs:
     runs-on: windows-latest
```

Comment:
Nice, it looks like the test passed.

Add comment and/or change the workflow name as well so we remember that this unit test includes Schannel SSL backend.